Create folder design with all mockups (#54)
KrispyTech/airneis: docs/design/mockups/backoffice/layout.md

# Layout Mockup

![Layout Mockup](./images/layout.png)

> This is the main layout that will be used for all other pages in Backoffice.

Reviewer: ```suggestion
This is the main layout that will be used for all other pages in our back office.
```

> We can see the header, which contains "ÀIRNEIS" and "BACKOFFICE".

Reviewer: ```suggestion
We can see the header, which contains "ÀIRNEIS" and "Back Office".
```

Should be named : "Back Office"

> There is also a lateral bar which contains all the Backoffice navigations

Reviewer: ```suggestion
There is also a lateral bar which contains all of our back office navigations
```

> The first part of the lateral bar is the "Product Management":

Reviewer: ```suggestion
The first part of the lateral bar is for "Product Management":
```

- Products, to see all products in a table and manage them.
- Categories, to see all categories in a table and manage them.
- Higlanders, to manage "The Highlanders of the moment" in a table and choose the order in which they appear.

> - Orders, to manage the orders in a table and manage them

Reviewer: Can you use ": " instead of "," commas. 

Example : `Products: To see all products in a table and manage them. `

Author: Yes, I'm going to do it now.

> The second part of the lateral bar is the "User Management":

Reviewer: ```suggestion
The second part of the lateral bar is for "User Management":
```

- Users, to manage all users in a table that have a connection on the web or mobile.

> - Contacts, to manage forms written by the users.

Reviewer: Same as above, use ":" please. Thanks

Author: Yes, I'm going to do it now.

# Layout Mockup

![Layout Mockup](./images/layout.png)

This is the main layout that will be used for all other pages in our back-office.
We can see the header, which contains "ÀIRNEIS" and "Back Office".

There is also a lateral bar which contains all back office navigations

> <<<<<<< Updated upstream

Reviewer 2: why so much <<<<<<<<< ? Does it comes from a git conflict?
```suggestion

```

The first part of the lateral bar is for "Product Management":
- Products, to see all products in a table and manage them.
- Categories, to see all categories in a table and manage them.
- Higlanders, to manage "The Highlanders of the moment" in a table and choose the order in which they appear.
- Orders, to manage the orders in a table and manage them

The second part of the lateral bar is for "User Management":
- Users, to manage all users in a table that have a connection on the web or mobile.
- Contacts, to manage forms written by the users.

> =======

Reviewer 2: Remove

The first part of the lateral bar is the "Product Management":
- Products: To see all products in a table and manage them.
- Categories: To see all categories in a table and manage them.
- Higlanders: To manage "The Highlanders of the moment" in a table and choose the order in which they appear.
- Orders: To manage the orders in a table and manage them

The second part of the lateral bar is the "User Management":
- Users: To manage all users in a table that have a connection on the web or mobile.
- Contacts: To manage forms written by the users.

> >>>>>>> Stashed changes

Reviewer 2: Remove
